src/ocm/selectors/clusterSelectors.ts

```diff
@@ -4,7 +4,7 @@ import { Cluster } from '../../common/api/types';
 export const selectMachineNetworkCIDR = ({
   machineNetworks,
   machineNetworkCidr,
-}: Partial<Cluster>) => _.head(machineNetworks)?.cidr ?? machineNetworkCidr;
+}: Partial<Cluster>) => machineNetworkCidr ?? _.head(machineNetworks)?.cidr;
```

Comment:
I introduced this change, this is the way it should work. First attempt to grab the first machine network CIDR, but if it fails use the old `machineNetworkCidr` field
```suggestion
}: Partial<Cluster>) => _.head(machineNetworks)?.cidr ?? machineNetworkCidr;
```

Reply:
w/o the mock you shouldn't get the same result